app.rb

```diff
@@ -36,7 +36,11 @@ class Makersbnb < Sinatra::Base
      redirect ('/sessions/new')
    end
     session[:user_id] = user.id
-  
+  end
+
+  get '/logout' do
```

Comment:
We can use the truthy/falsy nature of User.find to do osme heavy lifting for us Add this code to all get routes that will involve a user.

@user = User.find(id: sessions['user_id']